model/QueueBroker/RdsQueueBroker.php

```diff
@@ -28,6 +28,7 @@
 use Doctrine\DBAL\Schema\SchemaException;
 use Exception;
 use InvalidArgumentException;
+use Google\Cloud\Spanner\Transaction;
```

Comment:
i think we should not do this, connection the Rds with the Cloud\Spanner is not the best option

Reply:
You're right, but this is only for the Spanner custom branch. Of course, this will be in an abstraction class for transaction, of course.